less/index.less

```diff
@@ -54,6 +54,192 @@ table {
 }
 
 html, body {
```

Comment:
Some of these properties (like max-width, margin) might be better on your container div